2d/pong_mcru/level/level.gd

```diff
@@ -0,0 +1,46 @@
+extends Node
+
+var player_score = 0
+var opponent_score = 0
+
+
+func score_achieved(score):
+	"""
+	Stops the ball, starts the count down and sums one into the score.
+	"""
+	get_tree().call_group("BallGroup", "stop_ball")
+	$CountDownLabel.visible = true
+	$CountDownTimer.start()
+	return score + 1
+
+
+func _on_PlayerGoal_body_entered(body):
+	"""
+	Checks if Opponent makes point
+	"""
+	opponent_score = score_achieved(opponent_score)
+
+
+func _on_OpponentGoal_body_entered(body):
+	"""
+	Checks if Player makes point
+	"""
```

Comment:
(1) I think that Godot reserved methods like `_ready()`, `_physics_process()` or signals in code(like this one) don't need docstrings, as they are already documented in the Godot docs. 

This is an interesting article around this topic https://refactoring.guru/smells/comments